changelog/2022-04-19T12_26_48-05_00_add_scanl1

```diff
@@ -0,0 +1 @@
+CHANGED: Add scanl1 for vectors to the prelude
```

Comment:
```suggestion
CHANGED: Before this release, `scanl1` was re-exported from the Haskell Prelude. Clash's Prelude now exports a `Vec` specialized version.
```

Write changelogs with the user reading it in mind. (I know we haven't applied this mantra consistently in the past, but I always end up spending a bit of time rewriting changelogs upon release.)

Reply:
Yes! Good point.